modal.css

```diff
@@ -0,0 +1,161 @@
+:root {
+  --space-cadet: #172b4d;
+  --black-olive: #3c3a39;
+  --police-blue: #344563;
+  --blaze-orange: #ff6b00;
+  --white: #fff;
+  --spanish-gray: #979493;
+  --cool-grey: #8993a4;
+  --citrine-brown: #943e00;
+  --font-family-1: roboto;
+  --font-family-2: inter;
+  --transition: all 0.5s ease-in;
+}
+
+body {
+  position: relative;
+}
+
+.cover-modal {
+  position: absolute;
+  top: 0;
+  margin: 0 1.4rem;
+  padding: 20px 30px 20px;
+  z-index: 10;
+  background-color: var(--white);
+}
+
+.heading-modal {
+  display: flex;
+  flex-direction: column;
+}
+
+.heading-modal h3 {
+  font-family: var(--font-family-1);
+  font-size: 22px;
+  font-weight: 700;
+  color: var(--space-cadet);
+  margin-bottom: 3rem;
+}
+
+.heading-modal-ul {
+  display: flex;
+  list-style: none;
+  gap: 13px;
+}
+
+.heading-modal-ul li {
+  background: transparent;
+  padding: 0.5rem 0.5rem;
+  border: 1px solid var(--cool-grey);
+  color: var(--black-olive);
+}
+
+.modal-details {
+  margin-top: 2rem;
+}
+
+.modal-details-img {
+  max-width: 100%;
+}
+
```

Comment:
- [ ] Good job fixing the rendering of your pop-up on mobile screen - the overlay especially, as requested by the previous reviewer :clap: :clap: The only issue I see in this is that you seem to have changed something that is having a huge negative user experience with the pop up in the desktop view :confused: The image is not displaying in equal length with the text as it was before. See it here :point_down: 
|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/223854076-9c5a4586-67fd-4025-a136-58985953d34f.png)|![image](https://user-images.githubusercontent.com/99715640/223854160-e0005244-ad25-47e3-a49b-5d8da194aa25.png)|

- _**Kindly fix the anomaly :+1: **_

Reply:
I would appreciate it if you could help me with this. Thank you.

Reply:
**Resolved!**
Kindly clone to see changes made.


<img width="1440" alt="Screenshot 2023-03-09 at 10 19 11" src="https://user-images.githubusercontent.com/92754123/223977764-6ecee938-0066-4539-add6-a6c1f25035a7.png">